net-misc/tigervnc/tigervnc-1.12.0-r1.ebuild

```diff
@@ -171,10 +169,20 @@ src_install() {
 			rm -v "${ED}"/usr/$(get_libdir)/xorg/modules/extensions/libvnc.la || die
 		fi
 
-		newconfd "${FILESDIR}"/${PN}.confd ${PN}
-		newinitd "${FILESDIR}"/${PN}.initd ${PN}
+		newconfd "${FILESDIR}"/${PN}-${PV}.confd ${PN}
+		newinitd "${FILESDIR}"/${PN}-${PV}.initd ${PN}
+
+		# this file is here to document what I tested and what works
+		# TODO: remove it when done
+		if [ -f "${ED}"/etc/tigervnc/vncserver-config-defaults ]; then
+			cat "${FILESDIR}"/vncserver-config-defaults >> "${ED}"/etc/tigervnc/vncserver-config-defaults || die
+		fi
```

Comment:
This part should be removed (and the file in `files`).  I'm keeping it for now as bookmark.